test/exercise/fp/solution.rb

```diff
@@ -4,12 +4,18 @@ class << self
       # Обратиться к параметрам фильма можно так:
       # film["name"], film["rating_kinopoisk"], film["rating_imdb"],
       # film["genres"], film["year"], film["access_level"], film["country"]
-      def rating(_array)
-        0
+      def rating(array)
+        films_sort = array.select do |film|
+          film['rating_kinopoisk'].to_f.positive? &&
```

Comment:
need to handle case where `film['rating_kinopoisk']` is `nil` or cannot be parsed to `float`
hint: safe navigation